src/mods/askUserForParam.ts

```diff
@@ -151,7 +153,11 @@ export async function ask(
   const result: any = {};
   const promises = _.map(substituted, (paramSpec: Param) => {
     return async () => {
-      result[paramSpec.param] = await askForParam(paramSpec);
+      if (lockedLocation && paramSpec.param === "LOCATION") {
+        result.LOCATION = lockedLocation;
```

Comment:
Do we want to add a log message here saying: "LOCATION is ${lockedLocation}" or something to that effect?

Reply:
Yes, please add a log saying: 

"Location is currently <location>, this cannot be modified. Please uninstall and reinstall the mod to change location. "

Reply:
Added